views/partials/vpn-banner.hbs

```diff
@@ -0,0 +1,52 @@
+<aside class="vpn-banner">
+	<section class="vpn-banner-top">
+		<label class="short-location">{{{ getString "vpn-banner-location" }}} <output></output></label>
+		<label>
+			<span class="protected-txt">{{{ getString "vpn-banner-protected-with-vpn" }}}</span>
+			<span class="protected-txt">{{{ getString "vpn-banner-protect-yourself-with-vpn" }}}</span>
+		</label>
+		<label class="vpn-banner-chevron" data-expand={{ getString "vpn-banner-cta-expand" }} data-close={{
+			getString "vpn-banner-cta-close" }}></label>
+	</section>
+	<section class="vpn-banner-bottom">
+		<h2 class="vpn-banner-title">
+			<span class="protected-txt">{{ getString "vpn-banner-title-1" }}</span>
+			<span class="protected-txt">
+				{{ getString "vpn-banner-title-2" }}
+				<span class="subheading">{{ getString "vpn-banner-subtitle-2" }}</span>
+				</span>
```

Comment:
Another bit of weird indentation that doesn't seem to match where it should be?

Reply:
fixed.